glide.yaml

```diff
@@ -52,4 +52,4 @@ import:
 - package: github.com/openshift/origin
   version: v3.11.0
   subpackages:
-  - pkg/oc/cli/login
+  - pkg/oc/cli/login
```

Comment:
since you updated `glide.yaml`, did you run `make vendor-update` after that ?
ideally there should be `glide.lock` and updated `vendor` directory as well

Reply:
Hmm, that's a whitespace. No dependencies were changed.

Reply:
Also, since many dependencies are not at a fixed version, randomly running `make vendor-update` will almost certainly result in changes in dependencies even without a change in `glide.yaml`…

Reply:
every dependency should be fixed to specific version. Only one that is currently not is github.com/hinshun/vt10x

Reply:
okay cool